### ISSUE TITLE
[bitnami/keycloak] keycloakConfigCli won't run

### ISSUE THREAD

**Issue by macmenco** [bitnami/charts#19826]
### Name and Version

bitnami/keycloak 17.0.1

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Creating a kyecloak with the following values gives an error in the keycloakConfigCli

```
2023-10-06 15:07:12.702 ERROR 1 --- [           main] d.a.k.config.KeycloakConfigRunner        : Could not connect to keycloak in 120 seconds: jakarta.ws.rs.ProcessingException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "features" (class org.keycloak.representations.info.ServerInfoRepresentation), not marked as ignorable (15 known properties: "protocolMapperTypes", "providers", "identityProviders", "themes", "passwordPolicies", "clientInstallations", "memoryInfo", "enums", "socialProviders", "clientImporters", "profileInfo", "componentTypes", "systemInfo", "cryptoInfo", "builtinProtocolMappers"])
 at [Source: (org.jboss.resteasy.specimpl.AbstractBuiltResponse$InputStreamWrapper); line: 1, column: 1200] (through reference chain: org.keycloak.representations.info.ServerInfoRepresentation["features"])
```

```
auth:
  adminUser: admin

proxy: edge

production: true 

replicaCount: 1

ingress:
  enabled: true
  ingressClassName: traefik
  hostname: sso.blabla.internal
  annotations:
    traefik.ingress.kubernetes.io/router.tls: "true"
    traefik.ingress.kubernetes.io/router.entrypoints: internalsecure
    kubernetes.io/ingress.provider: traefik
  tls: true

resources:
  limits: 
    cpu: 1
    memory: 512Mi

postgresql:
  primary:
    containerSecurityContext:
      enabled: false
    resources:
      limits: 
        cpu: 1
        memory: 256Mi

tls:
  enabled: false

metrics:
  enabled: true
  serviceMonitor:
    enabled: true

logging:
  level: WARN

keycloakConfigCli:
  enabled: true
  existingConfigmap: keycloak-config
  cleanupAfterFinished:
    enabled: true
    seconds: 600
```

### Are you using any custom parameters or values?

See values

### What is the expected behavior?

A running keycloak, configred according to the config in configmap

### What do you see instead?

The error

**Comment by javsalgar** [comment]
Hi,

Could you share the configmap configuration to see if this is something reproducible?

**Comment by tanadeau** [comment]
See https://github.com/adorsys/keycloak-config-cli/issues/927 and https://github.com/adorsys/keycloak-config-cli/pull/928

**Comment by macmenco** [comment]
Sure. Here's the configmap. But I saw the links to adorsys, so things should be fixed soon 😃 

```
{
  "enabled": true,
  "realm": "rlm",
  "displayName": "RLM",
  "defaultSignatureAlgorithm": "RS256",
  "sslRequired": "external",
  "registrationAllowed": false,
  "registrationEmailAsUsername": false,
  "userManagedAccessAllowed": true,
  "rememberMe": true,
  "verifyEmail": false,
  "loginWithEmailAllowed": true,
  "duplicateEmailsAllowed": false,
  "resetPasswordAllowed": true,
  "editUsernameAllowed": false,
  "bruteForceProtected": true,
  "permanentLockout": false,
  "maxFailureWaitSeconds": 900,
  "minimumQuickLoginWaitSeconds": 60,
  "waitIncrementSeconds": 60,
  "quickLoginCheckMilliSeconds": 1000,
  "maxDeltaTimeSeconds": 43200,
  "failureFactor": 30,
  "users": [
    {
      "username": "jdoe",
      "email": "j.doe@some.mail",
      "enabled": true,
      "firstName": "John",
      "lastName": "Doe",
      "emailVerified": true,
      "realmRoles": ["gitlab_users", "gitlab_admins"],
      "clientRoles": {
        "realm-management": ["realm-admin"]
      }
    }
  ],
  "roles": {
    "realm": [
      { "name": "gitlab_auditors" },
      { "name": "gitlab_users" },
      { "name": "gitlab_admins" },
      { "name": "gitlab_externals" }
    ]
  },
  "clients": [
    {
      "clientId": "gitlab",
      "name": "GitLab",
      "description": "GitLab",
      "enabled": true,
      "clientAuthenticatorType": "client-secret",
      "secret": "my-special-client-secret",
      "redirectUris": ["*"],
      "webOrigins": ["*"],
      "defaultClientScopes": [
        "web-origins",
        "acr",
        "profile",
        "email",
        "gitlab_roles"
      ],
      "optionalClientScopes": [
        "address",
        "phone",
        "roles",
        "offline_access",
        "microprofile-jwt"
      ]
    }
  ],
  "clientScopes": [
    {
      "name": "gitlab_roles",
      "description": "Realm roles for Gitlab",
      "protocol": "openid-connect",
      "attributes": {
        "include.in.token.scope": "true",
        "display.on.consent.screen": "true",
        "consent.screen.text": "${rolesScopeConsentText}"
      },
      "protocolMappers": [
        {
          "name": "realm roles",
          "protocol": "openid-connect",
          "protocolMapper": "oidc-usermodel-realm-role-mapper",
          "consentRequired": false,
          "config": {
            "multivalued": "true",
            "userinfo.token.claim": "true",
            "user.attribute": "foo",
            "id.token.claim": "true",
            "access.token.claim": "false",
            "claim.name": "roles",
            "jsonType.label": "String"
          }
        }
      ]
    }
  ],
  "defaultDefaultClientScopes": [
    "profile",
    "email",
    "web-origins",
    "acr",
    "gitlab_roles"
  ],
  "defaultOptionalClientScopes": [
    "offline_access",
    "address",
    "phone",
    "microprofile-jwt",
    "roles",
    "role_list"
  ],
  "internationalizationEnabled": true,
  "supportedLocales": ["en", "nl"],
  "defaultLocale": "en"
}
```

**Comment by tanadeau** [comment]
keycloak-config-cli 5.9.0-22.0.4 is out now

**Comment by javsalgar** [comment]
Hi,

Should this configuration be compatible with the latest keycloak-config-cli release?

**Comment by tanadeau** [comment]
Not to answer for @macmenco, but from my understanding, every configuration valid for CLI 5.8.0 and Keycloak 22.0.3 should be valid for CLI 5.9.0 and Keycloak 22.0.4.

**Comment by tanadeau** [comment]
I think this can be closed since version 17.1.1 was released with the upgrade to config cli 5.9.0.

**Comment by javsalgar** [comment]
Thanks for confirming!!
